src/pages/physical-count-itemDetails/index.js

```diff
@@ -0,0 +1,633 @@
+      extension: SCRepository.StockCountControllerTab.get,
+      parameters: parameters
+    })
+
+    if (resp) {
+      formik.setFieldValue('status', resp.record.status)
+    }
+  }
+
+  async function getDTDsku(stockCountId) {
+    let dtId
+    let disableSKULookup = false
+
+    const res = await getRequest({
+      extension: SCRepository.StockCount.get,
+      parameters: `_recordId=${stockCountId}`
+    })
+
+    console.log(dtId)
+    console.log(res)
+    console.log('resss', res?.record?.dtId)
+    res?.record?.dtId ? (dtId = res?.record?.dtId) : null
+    console.log(dtId)
+
+    if (dtId) {
+      const DTDres = await getRequest({
+        extension: SCRepository.DocumentTypeDefaults.get,
+        parameters: `_dtId=${dtId}`
+      })
+      console.log(DTDres)
+      DTDres?.record?.disableSKULookup ? (disableSKULookup = DTDres?.record?.disableSKULookup) : false
+    }
+
+    console.log(disableSKULookup)
+    setDisSkuLookup(disableSKULookup)
+  }
+  async function getSysChecks() {
+    const Jres = await getRequest({
+      extension: SystemRepository.SystemChecks.get,
+      parameters: `_checkId=${SystemChecks.POS_JUMP_TO_NEXT_LINE}&_scopeId=1&_masterId=0`
+    })
+
+    if (Jres?.record?.value) setJumpToNextLine(Jres?.record?.value)
+
+    const DQres = await getRequest({
+      extension: SystemRepository.SystemChecks.get,
+      parameters: `_checkId=${SystemChecks.DEFAULT_QTY_PIECES}&_scopeId=1&_masterId=0`
+    })
+
+    if (DQres?.record?.value) setShowDefaultQty(DQres?.record?.value)
+  }
+
+  const defaultQty = !jumpToNextLine && !showDefaultQty ? 0 : 1
+
+  const columns = [
+    {
+      component: disSkuLookup ? 'textfield' : 'resourcelookup',
+      name: 'sku',
+      label: _labels.sku,
+      props: {
+        ...(!disSkuLookup && {
+          endpointId: InventoryRepository.Item.snapshot,
+          mapping: [
+            { from: 'recordId', to: 'itemId' },
+            { from: 'sku', to: 'sku' },
+            { from: 'name', to: 'itemName' },
+            { from: 'priceType', to: 'priceType' }
+          ],
+          displayField: 'sku',
+          valueField: 'recordId',
+          columnsInDropDown: [
+            { key: 'sku', value: 'sku' },
+            { key: 'name', value: 'Name' },
+            { key: 'flName', value: 'FL Name' }
+          ],
+          displayFieldWidth: 1
+        }),
+        jumpToNextLine: jumpToNextLine
+      },
+      async onChange({ row: { update, newRow } }) {
+        let itemId
+        itemId = newRow?.itemId
+
+        // if (!disSkuLookup) {
+        //   itemId = newRow?.itemId
+        //   if (disableItemDuplicate) {
+        //     if (formik.values.rows.find(item => item.itemId == itemId)) {
+        //       update({
+        //         itemId: null,
+        //         itemName: '',
+        //         sku: null,
+        //         priceType: null
+        //       })
+        //     }
+        //     return
+        //   }
+        // }
```

Comment:
is this needed?

Reply:
yes we still have modifications in the screen so some code is commented out on purpose